apps/datahub/src/app/router/datahub-router.service.ts

```diff
@@ -76,6 +76,6 @@ export class DatahubRouterService {
   }
 
   getSearchRoute(): string {
-    return `${ROUTER_ROUTE_HOME}/${ROUTER_ROUTE_SEARCH}`
```

Comment:
It's quite curious that this always worked

Reply:
Yep, maybe due to the redirect in lines https://github.com/geonetwork/geonetwork-ui/pull/850/files#diff-18147f3d9fd25936e42e709dea85cb97a42c69b44e19d92cde9d43f409ad6baaR30-R34 above...